apps/backend/src/allowlist/pipes/sign-message.pipe.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  PipeTransform,
+  Injectable,
+  ArgumentMetadata,
+  BadRequestException,
+} from '@nestjs/common';
+import { StdSignature, verifyNonceMsgSigner } from 'cudosjs';
+import { SignedMessageDto } from '../dto/signed-message.dto';
+
+@Injectable()
+export class SignMessagePipe implements PipeTransform {
+  async transform(value: SignedMessageDto, metadata: ArgumentMetadata) {
+    if (
+      !value.signature ||
+      !value.address ||
+      !value.message ||
+      !value.sequence ||
+      !value.account_number ||
+      !value.chain_id
+    ) {
+      throw new BadRequestException('Missing data');
```

Comment:
woulnd't it be good to have a more detailed message on which data exactly is missing here